docs/README.md

```diff
@@ -28,3 +36,69 @@
 - [Writing performant React components](writing-performant-react-components.md)
 - [Deployment](deployment.md)
 - [Debugging](debugging.md)
+
+## Tools used
+
+- **pnpm**: a package manger
+  - pnpm's workspace feature is the foundation of the monorepo
+- **turborepo**: a monorepo scripting tool
+  - parallelize script execution
+  - manage execution dependency
+  - cache outputs to accelerate execution
+- **syncpack**: a monorepo dependency manager
+  - synchronize package dependencies
+  - validate dependency version ranges with configurable rules
+  - format and lint package json
+- **changeset**: a monorepo version manager
+  - increment semver in a topological way
+  - progressively compile release notes as PRs merge
+- **vite**: a bundler/dev server
+  - vite bundles monorepo apps for deployment
+  - vite's lib mode also builds and bundles some packages
+- **vitest**: a testing framework
+  - vitest is workspace-aware
+  - testing and bundling can use the same vite config
+  - vitest provides in-browser testing via playwright
+
+## The Basics
+
+### Local, compact chain state
```

Comment:
This is all GREAT. This is the type of stuff I wish was in the README when I first joined 👏🏻